climada/test/test_plot.py

```diff
@@ -169,6 +179,77 @@ def test_disc_rates(self):
         rates[95:120] = 0.035
         disc = DiscRates(years=years, rates=rates)
         disc.plot()
+
+    def test_cost_benefit(self):
+        """ Test plot functions of cost benefit"""
+        
+        """Hazard."""
```

Comment:
Please do not use docstrings for commentating code. Docstrings go right below the function/class definition. Use regular comments everywhere else

```suggestion
        # Load hazard from the data API
```